.rspec

```diff
@@ -1,5 +1,2 @@
---colour
```

Comment:
Why remove these lines?

Reply:
* `--colour` is now on by default ([link](https://github.com/rspec/rspec-core/blob/master/Changelog.md#360beta1--2016-10-09)

Reply:
OK, makes sense.